Tools/Apply_Link_Check.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Script taken from https://github.com/OWASP/CheatSheetSeries/blob/master/scripts/Apply_Link_Check.sh
+# Script in charge of auditing the released MD files in order to detect dead links
+cd ../Document
+find . -name \*.md -exec markdown-link-check -q -c ../.markdownlinkcheck.json {} \; 1>../link-check-result.out 2>&1
```

Comment:
That's how you found the dead links! Great idea, thanks for putting into the repo